### PR TITLE
Make popperDelay overridable

### PR DESCRIPTION
**Pull request by alexandremoore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,6 +192,7 @@ NgModule({
   
    | Options                  | Type              | Default  |
    |:-------------------      |:----------------  |:-------- |
+   | showDelay                | number            | 0    |
```
Comment (MrFrankel) [inline README.md]: align right table line with table

**Comment by MrFrankel** [comment]
@alexandremoore can you resolve to conflict so I can merge?

**Comment by alexandremoore** [comment]
@MrFrankel, sorry I was out all weekend. Thanks for merging the PR.

**Comment by MrFrankel** [comment]
@alexandremoore No prob, thanks for the PR.
released in 2.5.0

**Pull request merged**